miasm2/jitter/codegen.py

```diff
@@ -283,6 +283,8 @@ def traverse_expr_dst(self, expr, dst2index):
                         "0")
 
         else:
+            # Spot case like Cond?X:Y
+            assert -1 not in dst2index.values()
```

Comment:
Well, the value is erased in `dst2index`. But the keys of `dst2index` are further used to generate the goto (in some case). I'm not completely sure that there is no way that this impact the resulting `switch`. There are several conditions to consider (ExprInt in or out of `instr_offsets`, label, ExprId, ...) in `gen_dst_goto`, `gen_goto_code` and in this method.
If you are sure about it, I will remove this commit.